orb/commands/@drupal.yml

```diff
@@ -103,6 +103,19 @@ drupal-helm-deploy:
         name: Deploy helm release
         no_output_timeout: "15m"
         command: |
+          #Detect pods in FAILED state
+          function failing_pods() {
```

Comment:
I would make sure the function name contains a verb, for example `list_failing_pods()` or `show_failing_pods()`, `failing_pods` seems to be a variable, especially for people less familiar with bash syntax.